zio-k8s-client/src/main/scala/com/coralogix/zio/k8s/client/config/KeyManagers.scala

```diff
@@ -34,12 +34,12 @@ private object KeyManagers {
     certificate: KeySource,
     key: KeySource,
     password: Option[String]
-  ): ZIO[System, Throwable, Array[KeyManager]] =
+  ): ZIO[Any with Scope, Throwable, Array[KeyManager]] =
```

Comment:
No need to leak the scope if you do the changes I described above

Reply:
done.